pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -298,7 +298,7 @@ func Run(ps *pubsub.PubSub) {
 		DNSctx.usableAddressCount)
 
 	// Timer for deferred sends of info messages
-	deferredChan := zedcloud.InitDeferred()
+	deferredChan := zedcloud.InitDeferred(agentName)
```

Comment:
This is useful, but I think it would make sense to split it off as a separate PR.
(I need this to complete the source in the logs and might refactor it a bit which would be a local change in zedcloud package.)

Reply:
https://github.com/lf-edge/eve/pull/1318